.github/workflows/lint.yml

```diff
@@ -32,8 +32,18 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v2
 
+        # Use custom problem matchers for better GH annotations
+      - name: Remove old matchers
+        run: echo "::remove-matcher owner=rust::"
+
+      - name: Add problem matchers
+        run: echo "::add-matcher::.github/problemMatchers/rust.json"
+
       - name: Run cargo fmt
-        run: cargo fmt --all -- --check
+        uses: actions-rs/cargo@v1
+        with:
+          command: fmt
+          args: --all -- --check
```

Comment:
we are doing the simple run here because cargo is preinstalled on the actions runners, which saves the time it takes to install the action

Reply:
okay, will revert this